src/term/flavour_py/mod.rs

```diff
@@ -0,0 +1,98 @@
+mod order_enums;
+pub mod to_lang;
+
+use indexmap::IndexMap;
+use interner::global::GlobalString;
+
+use crate::term::Name;
+
+use super::Op;
+
+#[derive(Clone, Debug)]
+pub enum Term {
+  // "None"
+  None,
+  // [a-zA-Z_]+
+  Var { nam: Name },
+  // [0-9_]+
+  Num { val: u32 },
+  // {fun}(args,)
+  Call { fun: Box<Term>, args: Vec<Term> },
+  // "fun" {pat} -> {bod}
```

Comment:
Although i prefer this, if we're imitating python, I think we should try to go all in and use `lambda x1 ... xn: term`.